bin/bin.js

```diff
@@ -355,6 +379,9 @@ const main = async () => {
 
     nextItem();
   }
+  if (execErrors != 0) {
```

Comment:
Let's just log and then ignore the exec errors for now. We can remove these lines.

Reply:
Okay I reverted https://github.com/lightpohl/podcast-dl/commit/470c2f1b6919dda2da451f46c6b251feec3d2859. So no error code any more for exec errors, just logging.